### ISSUE TITLE
NVDA screen reader is announcing unnecessary information as "press ALT + F1 key to expand accessibility option" but nothing is happening when we hit these keys:A11y_MonacoEditor_Screenreader

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/monaco-editor#4626]
### GitHubTags:
#A11yTCS; #A11ySev2; A11yMAS; #WCAG1.3.1; #Desktopweb; #Win11; #Visual Studio Code Client; #VisualStudioCode-Web-Apr2021; #NVDA; #JAWS; #Chrome; #a11y-monacoeditor; #SCIM:bug28133995;

### Environment Details:
#URL: [Monaco Editor (microsoft.github.io)](https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0-dev.20230707#XQAAAAJzAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscwzfibW_1uhs0-3YGTBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKimARHSWlrTPOQHROITHxmAetfeRrm3VHdSe7_Etv6WpgzlH4vE8EOD3EPhJDl6tDpOO8RxKZbUytKns3T0NFFloo9Q21KsEBT298I6cyZc5-Glup9UHU6QTpv2Hjab6pZzB3ZO2RH-WGhzXse2-hNlgz6zCgsG8AjiqGg0ZzWzND7z09NmCQOQd6pfkNvO2zmMzFvHpTY8RFEy1urlQcQMb1HHLEESBi8m3Rr3UzFlKzY51hfYcz__QCePc)
Application name: Monaco Editor
Chrome: Version 127.0.6533.73 (Official Build) (64-bit)
Windows 11 Enterprise (Version 23H2)  
OS Build 22631.3737
Screen reader: NVDA 2024.2, JAWS

### Repro Steps:

1. Start NVDA/ JAWS screen reader.
2. Hit the URL: https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0-dev.20230707#XQAAAAJzAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscwzfibW_1uhs0-3YGTBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKimARHSWlrTPOQHROITHxmAetfeRrm3VHdSe7_Etv6WpgzlH4vE8EOD3EPhJDl6tDpOO8RxKZbUytKns3T0NFFloo9Q21KsEBT298I6cyZc5-Glup9UHU6QTpv2Hjab6pZzB3ZO2RH-WGhzXse2-hNlgz6zCgsG8AjiqGg0ZzWzND7z09NmCQOQd6pfkNvO2zmMzFvHpTY8RFEy1urlQcQMb1HHLEESBi8m3Rr3UzFlKzY51hfYcz__QCePc.
3. TAB to Monaco editor.
4. Observe that when we navigate to editor, screen reader is announcing as "press ALT + F1 key to expand accessibility option".

### Actual Results:
NVDA screen reader is announcing unnecessary information as "press ALT + F1 key to expand accessibility option" but nothing is happening when we hit these keys.

**Similar issue is also observed with Chrome + JAWS.**

### Expected Results:
NVDA screen reader should not be announcing unnecessary information as "press ALT + F1 key to expand accessibility option". If these keys are valid then when we hit ALT + F1 key, accessibility options should be available for users.

### User Impact:
Screen reader users will get incorrect information here, as he will try to execute these keys and no action will be performed which may lead to frustration for user.

### Attachment:

https://github.com/user-attachments/assets/616b8f64-6042-4dfd-8b3b-e93a35d63ea1

[4626_JAWS attachment.webm](https://github.com/user-attachments/assets/83df7183-07e4-4ba3-8f3e-2758071c6d6a)

**Comment by PoojaNamde** [comment]
Closing this bug as logged new bug to track this https://github.com/microsoft/monaco-editor/issues/4674

#Closed; #Regressed:09-09-24;
